demos/defender/aws_iot_demo_defender.c

```diff
@@ -268,27 +278,29 @@ int RunDefenderDemo( bool awsIotMqttMode,
     if( status == EXIT_SUCCESS )
     {
         /* Create MQTT Connection */
-        mqttStatus = _establishMqttConnection( pIdentifier,
-                                               pNetworkServerInfo,
-                                               pNetworkCredentialInfo,
-                                               pNetworkInterface,
-                                               &mqttConnection );
+        status = _establishMqttConnection( pIdentifier,
+                                           pNetworkServerInfo,
+                                           pNetworkCredentialInfo,
+                                           pNetworkInterface,
+                                           &mqttConnection );
 
-        if( mqttStatus != IOT_MQTT_SUCCESS )
+        if( status != EXIT_SUCCESS )
         {
-            IotLogError( "Failed to Create MQTT Connection, error: %s", IotMqtt_strerror( mqttStatus ) );
+            IotLogError( "Failed to Create MQTT Connection with error: %s",
+                         IotMqtt_strerror( mqttStatus ) );
```

Comment:
The `mqttStatus` value should not be logged here as it was not received from `_establishMqttConnection`?

Reply:
Ah yes missed that thanks.